apps/backend/src/modules/categories/libs/types/score-request-data.type.ts

```diff
@@ -0,0 +1,5 @@
+import { type QuizScoreDto } from "./types.js";
+
+type ScoreRequestData = Omit<QuizScoreDto, "createdAt" | "id" | "updatedAt">;
```

Comment:
```suggestion
type ScoreRequestDto = Omit<QuizScoreDto, "createdAt" | "id" | "updatedAt">;
```
Create a new type, instead of using Omit, try to avoid omitting more than 1 field